spec/metric_fu/metrics/reek/generator_spec.rb

```diff
@@ -43,51 +43,63 @@
       MetricFu::Configuration.run {}
       allow(File).to receive(:directory?).and_return(true)
       @reek = MetricFu::ReekGenerator.new
+      @examiner = @reek.send(:examiner)
+      @smell_warning = Reek.const_defined?(:SmellWarning) ? Reek.const_get(:SmellWarning) : Reek.const_get(:Smells).const_get(:SmellWarning)
+      if @smell_warning.instance_methods.include?(:subclass)
```

Comment:
I'm not too happy with this kind of hacks in the specs, but I can't think of a better solution right now.
Ruby 1.9.2. is using an older reek version which doesn't implement SmellWarning#smell_type, due to which the verified double fails.
OTOH the older reek output is tested separately.

@bf4, maybe you have a better solution?